SDL/SDLJoystick.h

```diff
@@ -30,4 +30,5 @@ class SDLJoystick{
 	bool registeredAsEventHandler;
 	std::vector<SDL_GameController *> controllers;
 	std::map<int, int> controllerDeviceMap;
+	int _njoy;
```

Comment:
The convention we use (but not followed everywhere yet, as seen here) is to have a _ suffix, not prefix, for member variables.